snapshot/restore/local_fs_output.go

```diff
@@ -392,6 +392,10 @@ func write(targetPath string, r fs.Reader, size int64, c streamCopier) error {
 		return errors.Wrapf(err, "cannot write data to file %q", f.Name())
 	}
 
+	if err := f.Sync(); err != nil {
```

Comment:
How much longer would it take to restore a backup make of 100K small files (< 1KB each)?

Reply:
I don't have opportunity to benchmark it yet, so I don't have accurate data.
But I would say it would definitely increase the write time and it depends on the file system and storage system. In some cases the time may be remarkable.

On the other hand, I think this time should be included into the restore time --- literally we need to make sure the data is written to the storage before we say that restore completes. In this concept, this is an existing gap, this is why I didn't make it optional.

Here we can make an analogy:

- When restoring a volume from a CSI snapshot, we create a PVC with its `dataSource` pointing to the VolumeSnapshot, that is all
- But we cannot say the restore is done since the storage needs to clone a volume from it which would take a remarkable time

Reply:
It needs to be optional

Reply:
OK. Will add an option for it.